### PR TITLE
fix urllib3 errors, and added -y for installs

### PR DESCRIPTION
**Pull request by Tango43**

### CODE REVIEW

**Comment by QuelonaSec** [comment]
I tried the code of Tango43, but i still got errors...

I had to install requests:
`pip install requests`
and import it
`import requests`

But after all it stops with 

> No Hostnames Found
LIVE IPS FOUND! THERE IS NO NEED TO CONTINUE! WARBERRY WILL NOW EXIT!
Waiting for Responder to finish!!!

File: setup.sh [hunk 1]
```diff
@@ -7,6 +7,7 @@ apt-get install -y python-scapy
 apt-get install -y tcpdump
 apt-get install -y nmap
 apt-get install -y python-urllib3
+apt-get install -y cme
```
Comment (QuelonaSec) [inline setup.sh]: That's not working:
Package cme in raspbian sources (https://packages.debian.org/stretch/cme) is not what we're looking for...
In this context cme means CrackMapExec (https://github.com/byt3bl33d3r/CrackMapExec/wiki/Installation) which can be installed like so:
`#~ apt-get install -y libssl-dev libffi-dev python-dev build-essential
#~ sudo pip install crackmapexec`

Comment (QuelonaSec) [reply setup.sh]: Where did you spot this urllib3 errors?

Right now I'm testing the original code (in raspbian stretch) and I'm curious if your changes are necessary.

**Comment by secgroundzero** [comment]
fixed with installing python-requests

**Pull request closed without merging**